packages/vats/src/ibc.js

```diff
@@ -343,18 +354,29 @@ export const prepareIBCProtocol = (zone, powers) => {
 
           return kit.vow;
         },
-        async onListen(_port, localAddr, _listenHandler) {
+        /**
+         * @param {Port} _port
+         * @param {LocalIbcAddress} localAddr
```

Comment:
Implementation signatures of these callbacks, like `onListen`'s `localAddr` parameter, must be compatible with their interface signatures, specified in `packages/network/src/types.js`.  IBC is just the first of several possible supported network implementations.